src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -732,7 +737,7 @@ private void InvokeMarshaledCallbacks()
                 {
                     // Optimize a common case of using EventHandler. This allows us to invoke
                     // early bound, which is a bit more efficient.
-                    EventHandler c = current as EventHandler;
+                    EventHandler? c = current as EventHandler;
```

Comment:
Could be simplified by using `is` instead in the `if (c != null)` statement.

Reply:
Ok.